<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Student Group Database</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Database</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Year</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Degree</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Faculty</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Captain</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Students</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="statusLabel">
          <property name="text">
           <string>Total groups: 0</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Operations</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="loadButton">
            <property name="text">
             <string>Load Database</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveButton">
            <property name="text">
             <string>Save Database</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="sortButton">
            <property name="text">
             <string>Sort by Name</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="deleteButton">
            <property name="text">
             <string>Delete Selected</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="searchEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="searchButton">
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>                  Year Range:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="startYearSpin">
            <property name="minimum">
             <number>2000</number>
            </property>
            <property name="maximum">
             <number>2100</number>
            </property>
            <property name="value">
             <number>2020</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>                         to</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="endYearSpin">
            <property name="minimum">
             <number>2000</number>
            </property>
            <property name="maximum">
             <number>2100</number>
            </property>
            <property name="value">
             <number>2023</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="filterButton">
            <property name="text">
             <string>Filter</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLineEdit" name="facultyFilterEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="facultyButton">
            <property name="text">
             <string>Faculty Groups</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="splitButton">
            <property name="text">
             <string>Split by Degree</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Add Group</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Group Name:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="nameEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Year:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="yearSpin">
            <property name="minimum">
             <number>2000</number>
            </property>
            <property name="maximum">
             <number>2100</number>
            </property>
            <property name="value">
             <number>2023</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Degree:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="degreeCombo">
            <item>
             <property name="text">
              <string>Bachelor (B)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Specialist (C)</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Master (M)</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Faculty:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="facultyEdit"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Captain:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="captainEdit"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Student Count:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QSpinBox" name="countSpin">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>30</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="addButton">
          <property name="text">
           <string>Add Group</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
